test/rules_test.js

```diff
@@ -32,6 +32,12 @@ describe('rules', () => {
         ['x^(--(x + 1))', 'x^(x + 1)']
     ])
 
+    suite('rearrange coefficient', rules.REARRANGE_COEFF, [
+        ['y^3 * 5', '5 y^3'],
+        ['yz * 3', '3 yz'],
+        ['3x^2 * 5', '5 (3 x^2)']
```

Comment:
Yes, but it would also be pretty easy to also handle `y * z * 3`, `x * y * z * 3`, etc... note that `yz` is a single identifier.  As long as there is only one constant factor it's obvious what should be done.